src/utils/siteLaunchUtils.ts

```diff
@@ -0,0 +1,11 @@
+import { WHITE_LISTED_REPOS } from "constants/siteLaunch"
+
+export const isUserUsingSiteLaunchFeature = (siteName: string): boolean => {
```

Comment:
nit: `shouldUseSiteLaunchFeature` reads better and less verbish (flags should be nouns)

Reply:
a355257f56fd66c1d93c015a41fae38ef75594da

Reply:
a355257f56fd66c1d93c015a41fae38ef75594da